proposed/phpdoc-tags.md

```diff
@@ -641,16 +638,10 @@ function or method. When provided, it MUST contain a "Type"
 to indicate what is returned; the description on the other hand is OPTIONAL yet
 RECOMMENDED in case of complicated return structures, such as associative arrays.
 
-The @return tag MAY have a multi-line description and does not need explicit
-delimiting.
-
-It is RECOMMENDED to use this tag with every function and method. An exception to
-this recommendation, as defined by the Coding Standard of any individual project,
-MAY be:
-
-   **functions and methods without a `return` value**: the @return tag MAY be
-   omitted here, in which case an interpreter MUST interpret this as if
-   `@return void` is provided.
+It is RECOMMENDED to use this tag with every function and method.
+If no `@return` type is given, and no return type declaration is provided in
+the code signature, an interpreter MUST interpret this as if `@return mixed`
+is provided.
```

Comment:
So is the interpreter not allowed to look at native return type?

Reply:
Please re-read this sentence. It states also "when no return type is defined in the code signature" 😉